.github/CODEOWNERS

```diff
@@ -22,7 +22,7 @@
 # Service teams
 
 # PRLabel: %Communication
-/sdk/communication/                                 @vcolin7 @Azure/azure-sdk-write-communication
+/sdk/communication/                                 @vcolin7 @Azure/azure-sdk-communication-code-reviewers
 
 # PRLabel: %Communication - Common
 /sdk/communication/azure-communication-common/      @Azure/acs-identity-sdk @AikoBB @maximrytych-ms @mjafferi-msft
```

Comment:
just a call out it looks like github doesn't recognize this group: `@Azure/acs-identity-sdk` maybe we should just remove it?

Reply:
Hi @JoshuaLai ! We created this group a few years ago and we can take a look at it why it can't be recognized, and fix it either by removing or having a proper group